documenting.rst

```diff
@@ -1655,15 +1655,15 @@ in production; others are works in progress.
 +-----------------+-------------------------------+----------------------------+
 
 .. _article_pt_br: https://rgth.co/blog/python-ptbr-cenario-atual/
-.. _bpo_cocoatomo: https://bugs.python.org/user19001
```

Comment:
Please also change the prefix here from `_bpo_usernames` into `_gh_usernames`

Reply:
Done